src/popupbutton.cpp

```diff
@@ -63,6 +63,15 @@ void PopupButton::performLayout(NVGcontext *ctx) {
 
     const Window *parentWindow = window();
 
+	Widget* checkScroll = parent();
+	while (checkScroll != parentWindow){
```

Comment:
rather than checking for equality with parentWindow, do a loop until ``widget`` (rename variable to this) becomes ``nullptr``.